packages/leancode_hooks/lib/src/use_focused.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+
+bool useFocused(FocusNode focusNode) {
+  final focued = useState<bool>(false);
+
+  useEffect(
+    () {
+      void listener() {
+        focued.value = focusNode.hasFocus;
+      }
+
+      focusNode.addListener(listener);
+      return () => focusNode.removeListener(listener);
+    },
+    [focusNode],
+  );
+
+  return focued.value;
+}
```

Comment:
@Albert221 this one is yours 😝

Reply:
It seems they do the same. 

In this case I'd ditch this hook.

Reply:
IMO there is no need for a hook for this, `useListenable` covers this usecase (and it's what I have been using)